src/azure-cli/azure/cli/command_modules/containerapp/aaz/latest/network/__cmd_group.py

```diff
@@ -0,0 +1,20 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+class __CMDGroup(AAZCommandGroup):
```

Comment:
could we reuse aaz in network command modules?

Reply:
Confirm with Kai ru, aaz/network is for decouples it from the module code in the network.
It is recommended solution.